pkg/cmd/admin/policy/modify_scc.go

```diff
@@ -0,0 +1,234 @@
+package policy
+
+import (
+	"errors"
+	"io"
+
+	"github.com/spf13/cobra"
+
+	kapi "k8s.io/kubernetes/pkg/api"
+	kclient "k8s.io/kubernetes/pkg/client/unversioned"
+	kcmdutil "k8s.io/kubernetes/pkg/kubectl/cmd/util"
+	"k8s.io/kubernetes/pkg/util"
+
+	authorizationapi "github.com/openshift/origin/pkg/authorization/api"
+	"github.com/openshift/origin/pkg/cmd/util/clientcmd"
+	uservalidation "github.com/openshift/origin/pkg/user/api/validation"
+)
+
+const (
+	AddSCCToGroupRecommendedName      = "add-scc-to-group"
+	AddSCCToUserRecommendedName       = "add-scc-to-user"
+	RemoveSCCFromGroupRecommendedName = "remove-scc-from-group"
+	RemoveSCCFromUserRecommendedName  = "remove-scc-from-user"
+)
+
+type SCCModificationOptions struct {
+	SCCName      string
+	SCCInterface kclient.SecurityContextConstraintsInterface
+
+	DefaultSubjectNamespace string
+	Subjects                []kapi.ObjectReference
+}
+
+func NewCmdAddSCCToGroup(name, fullName string, f *clientcmd.Factory, out io.Writer) *cobra.Command {
+	options := &SCCModificationOptions{}
+
+	cmd := &cobra.Command{
+		Use:   name + " SCC GROUP [GROUP ...]",
+		Short: "Add groups to a security context constraint",
+		Long:  `Add groups to a security context constraint`,
+		Run: func(cmd *cobra.Command, args []string) {
+			if err := options.CompleteGroups(f, args); err != nil {
+				kcmdutil.CheckErr(kcmdutil.UsageError(cmd, err.Error()))
+			}
+
+			if err := options.AddSCC(); err != nil {
+				kcmdutil.CheckErr(err)
+			}
+		},
+	}
+
+	return cmd
+}
+
+func NewCmdAddSCCToUser(name, fullName string, f *clientcmd.Factory, out io.Writer) *cobra.Command {
+	options := &SCCModificationOptions{}
+	saNames := util.StringList{}
+
+	cmd := &cobra.Command{
+		Use:   name + " SCC USER [USER ...]",
+		Short: "Add users to a security context constraint",
+		Long:  `Add users to a security context constraint`,
+		Run: func(cmd *cobra.Command, args []string) {
+			if err := options.CompleteUsers(f, args, saNames); err != nil {
+				kcmdutil.CheckErr(kcmdutil.UsageError(cmd, err.Error()))
+			}
+
+			if err := options.AddSCC(); err != nil {
+				kcmdutil.CheckErr(err)
+			}
+		},
+	}
+
+	cmd.Flags().VarP(&saNames, "serviceaccount", "z", "service account in the current namespace to use as a user")
```

Comment:
> would be nice if the admin didn't have to switch namespaces to use the command but I'm ok as is if we're going to EOL these commands anyway (per Clayton's comments)

This effectively means using `-n sa-namespace`.  Since SCCs are cluster level, that should always be safe.